executor-service/src/test/java/executor/service/model/ThreadPoolConfigTest.java

```diff
@@ -0,0 +1,67 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class ThreadPoolConfigTest {
```

Comment:
Remove this 'public' modifier:  JUnit5 test classes and methods should have default package visibility